test/EndToEndTests/Common/Microsoft.OData.Client.E2E.TestCommon/TestPriorityAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="TestPriorityAttribute.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved.
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+namespace Microsoft.OData.Client.E2E.TestCommon
+{
+    public class TestPriorityAttribute : Attribute

```

Comment:
What does this attribute do? Maybe add a doc comment?

Reply:
It controls the order in which tests are executed.